client/realtor.lua

```diff
@@ -1,24 +1,28 @@
-
+local sharedConfig = require 'config.shared'
 local values = {}
-for k in pairs(Interiors) do
+for k in pairs(sharedConfig.interiors) do
     values[#values + 1] = k
 end
 
 local shell = 0
 local playerCoords = vec3(0, 0, 0)
 local isPreviewing = false
 
+local function calculateOffsetCoords(propertyCoords, offset)
+    return vec4(propertyCoords.x + offset.x, propertyCoords.y + offset.y, (propertyCoords.z - sharedConfig.shellUndergroundOffset) + offset.z, propertyCoords.w or 0.0)
+end
```

Comment:
Since the code is shared with the server, it should preferably be in a shared file to reduce mistakes. Either keep it in the config or create a shared file.